Sources/NIO/EventLoop.swift

```diff
@@ -596,6 +602,10 @@ extension EventLoop {
     public func preconditionInEventLoop(file: StaticString = #file, line: UInt = #line) {
         precondition(self.inEventLoop, file: file, line: line)
     }
+
+    public var isOpen: Bool {
```

Comment:
Can we mark this as the default implementation of this protocol requirement, and to be removed in 3.0? It's a bit of a nasty hack, and we'd like to avoid it being used if we can.

Reply:
Yes I agree with that, ideally we don’t have to do this at all.  Will do.

Reply:
@lukasa actually after reading your other comment, we may not need to change this protocol at all, if the `submit` function isn’t calling isOpen.  Perhaps this PR should just solve the issue in the context of MultithreadedEventLoop and not touch EventLoop protocol.

Reply:
Sounds like a good idea to me.